cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -77,6 +77,8 @@
   if they don't exist. */
   public static final String REQUIREMENTS_DATASET_TYPE_EXCLUDE = "requirements.datasetTypes.exclude.list";
 
+ //Holds a comma separated list of enabled accelerators
```

Comment:
Generally, don't add constants to the top level. Add it to the corresponding inner class (e.g. AppFabric), or add a new inner class as grouping. If you foresee there will be more configs for accelerator, maybe worth to add a new inner class.

Reply:
Added to AppFabric for now. Will add new class if more fields are required.